src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Security/DoNotDisableHttpClientCRLCheck.cs

```diff
@@ -105,8 +105,7 @@ class DoNotDisableHttpClientCRLCheck : DiagnosticAnalyzer
 
         private static readonly ImmutableHashSet<string> typeToTrackMetadataNames = ImmutableHashSet.Create<string>(
             WellKnownTypeNames.SystemNetHttpWinHttpHandler,
-            WellKnownTypeNames.SystemNetHttpHttpClientHandler,
-            WellKnownTypeNames.SystemNetHttpCurlHandler);
```

Comment:
Can you please merge latest master into this branch?

Reply:
Done